[Blog] created a Blog for Introduction to Layer5 Learning Resources

hirentimbadiya opened the pull request:
**Description**

This PR fixes #2323 

**Notes for Reviewers**


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

vishalvivekm commented:
👏 
@hirentimbadiya Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

hirentimbadiya commented:
> 👏 @hirentimbadiya Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

okay 👍

vishalvivekm commented on src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:
```diff
@@ -0,0 +1,98 @@
+title: "Introduction to Layer5 Learning Resources"
+subtitle: Discover Learning Paths of Layer5
+date: 2023-07-22 05:23:00 -0530
+author: Hiren Timbadiya
```
```suggestion
author: Layer5 Team
```

vishalvivekm replied:
@hirentimbadiya

l5io commented:
🚀 Preview for commit ca5d099dd1c9f0ffa5c2cf979694c92201c766f2 at: https://64be7e2b6fbae90506aafff8--layer5.netlify.app

l5io commented:
🚀 Preview for commit c9b3b566e50919a60cd5a7d641cdd1c25ad28ce4 at: https://64bfc2d4dd9b836459460039--layer5.netlify.app

thisiskaransgit commented on src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:
```diff
@@ -0,0 +1,98 @@
+</div>
+
+<h2>Conclusion</h2>
+<p>Layer5 is a creator and maintainer of service mesh standards and this makes it the best place for you to gain this knowledge.  One of our fantastic initiatives is the interactive practice labs, which let you study while you explore services meshes on your browser. As the primary developers of the cloud native management plane, Meshery, our maintainers are readily available to answer your quest ...
```
```suggestion
<p>Layer5 is a creator and maintainer of service mesh standards and this makes it the best place for you to gain this knowledge. One of our fantastic initiatives is the interactive practice labs, which let you study while you explore services meshes on your browser. As the primary developers of the cloud native management plane, Meshery, our maintainers are readily available to answer your questions, should in case you have any.
```

thisiskaransgit commented on src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:
```diff
@@ -0,0 +1,98 @@
+<p>Depending on your role as a developer, operator, or security engineer, you can choose the learning path most relevant to your everyday focus. This learning path currently has two focus areas for you to explore.</p>
+
+<ul>
+    <li><Link to="learn/learning-paths/mastering-service-meshes-for-developers">Mastering Service Meshes For Developers</Link> This introductory course in the Learning Path focuses on Developers and what they need to know to successfully design workloads for a service mesh environment.</li>
```
Correct the link here, please

thisiskaransgit commented on .github/workflows/meshmap.yml:
Revert this change, please

thisiskaransgit reviewed (commented):
@hirentimbadiya, thank you for working on this, here are some points to consider
- Keep "service mesh"  lowercased for all instances, except for the hyperlinks
- Remove the hyphen between cloud native

thisiskaransgit commented:
Also, we will be needing an updated thumbnail for the blog. // @ritiksaxena124

l5io commented:
🚀 Preview for commit 37a15b8ea671b2e19c2708a797ef59fd4927d499 at: https://64c1e5af0ae95f3a19a225b6--layer5.netlify.app

hirentimbadiya commented:
@thisiskaransgit, I have implemented some changes. Please review them, and  waiting for @ritiksaxena124 to upload the updated thumbnail image.

l5io commented:
🚀 Preview for commit 485516818e4ef8d5df05a28ac0c901d626ec046f at: https://64c1eceae696c643b867de9c--layer5.netlify.app

hirentimbadiya commented:
I have incorporated all the recommendations from @thisiskaransgit and have also updated the thumbnail provided by @ritiksaxena124. Please review this pull request, @vishalvivekm, @thisiskaransgit, @Chadha93.

l5io commented:
🚀 Preview for commit 6666ffbc6186d92d6ed8748587dcd418de682f25 at: https://64c496a7eececd34ce7cdfc5--layer5.netlify.app

l5io commented:
🚀 Preview for commit 6666ffbc6186d92d6ed8748587dcd418de682f25 at: https://64c496a974848a367803a61d--layer5.netlify.app

sudhanshutech commented:
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

l5io commented:
🚀 Preview for commit 9994f3ce2965ea76ca80ad8083336c6803c6da40 at: https://64c88ae1145ca43764a8f55a--layer5.netlify.app

hirentimbadiya commented:
@vishalvivekm , @thisiskaransgit kindly review this PR.

hirentimbadiya commented:
@ritiksaxena124 can you provide an image like this for blog ? 
![learning-paths-architechture](https://github.com/layer5io/layer5/assets/86219935/54a05ebb-d820-48ff-b71c-1e72757a0c2b)
